_posts/2019-11-14-april-2020-workshop.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: The First US-RSE Association Workshop
+posted_by: Ian Cosden
+tags: [workshop]
+---
+
+We are excited to announce that through a generous grant from the Alfred P. Sloan Foundation, we will be able to hold the first US-RSE Association community building workshop, April 21-22, 2020 in Princeton, NJ.
+This won't be an attempt to be an RSE conference, but a workshop focused on planning the best path forward to grow the US-RSE Association.
+
+
+The workshop will have breakout sessions focused on a number of different topics pertinent to the US-RSE community.
+We’ll be working together to generate material/content for the us-rse.org website that promotes and supports current/future RSEs in alignment with the [mission](https://us-rse.org/mission/) of the US-RSE Association, and to plan follow-up activities to grow the community.  
```

Comment:
Maybe just say content for the us-rse.org website, what's the difference between material and content?

Reply:
I would make this last bit a new sentence (it read weirdly in my head just now) E.g.,

> We’ll be working together to generate content for the us-rse.org website that promotes and supports current/future RSEs in alignment with the [mission](https://us-rse.org/mission/) of the US-RSE Association.  We will also plan follow-up activities to grow the community.

Reply:
Makes sense, I'll make the change.